### ISSUE TITLE
surface go: getting a usable font size

### ISSUE THREAD

**Issue by ThomasWaldmann** [jakeday/linux-surface#460]
when installing ubuntu 18.04.2 on the "go", the text is either:
- too small (100% scaling)
- too large (200% scaling)
- one can't set something in between via the ubuntu settings gui

This fixes at least the text size:
```
$ gsettings set org.gnome.desktop.interface text-scaling-factor 1.5
```

**Comment by qzed** [comment]
Gnome 3.32 on Wayland supports fractional scaling (at least the 1.5 factor, I think you can also enable this as experimental feature on previous versions). Apart from that you can try to play around with xrandr, although I've found that to not play too well with screen-rotation. From my experience, setting text-scaling is the best option on X11.

**Comment by jonas2515** [comment]
Yep, if you have GNOME 3.32 you should use fractional scaling on Wayland, it's still an experimental feature and you'll have to enable it using: `gsettings set org.gnome.mutter experimental-features "['scale-monitor-framebuffer']"`

This currently has one major drawback: Xwayland windows can't be scaled right now, so apps like Firefox or Chromium will look blurry. If you're using Firefox you can simply switch to Nightly, which uses Wayland natively.
